content/1_General/USACO_FAQs.mdx

```diff
@@ -238,6 +238,73 @@ or by googling. Answers to additional FAQs can be found
+
+## Q: Can I participate in multiple divisions concurrently?
+
+Yes, with a perfect score in your current division, you can be immediately promoted to the next division and participate in its contest during the same weekend.
+
+## Q: What types of problems can I expect in USACO?
+
+USACO problems cover a wide range of algorithmic challenges, requiring participants to demonstrate problem-solving skills and algorithmic thinking. These problems often involve tasks such as data manipulation, searching, sorting, dynamic programming, graph algorithms, and geometry. Participants are expected to devise efficient algorithms and implement solutions in their chosen programming language.
+
+## Q: What are the constraints on memory and time for C++, Python, and Java programs?
+In USACO contests, programs written in various programming languages are subject to specific constraints to ensure fair and efficient competition. These constraints include both time limits and memory allocations, which vary slightly depending on the language used.
+Time Constraints: Each input test case is allotted a specific time limit for program execution. This time limit typically ranges around 4 seconds, although it can vary depending on the complexity of the problem and the specific contest. Within this time frame, the program must complete its execution and produce the correct output. Exceeding the time limit results in the program being terminated, potentially leading to a partial or failed evaluation of the solution.
+
+- Memory Constraints: Programs are allocated a certain amount of memory for their execution. The allocated memory varies depending on the programming language and the contest specifications. For example, Java programs are typically allocated around 256 MB of memory, while other languages may have different memory allocations. The allocated memory is used to store variables, data structures, and other program-related information during execution. It's essential for participants to manage memory efficiently to avoid exceeding the allocated memory limit, which can lead to runtime errors or program termination.
+
+Here's a table summarizing the constraints for different programming languages in USACO contests:
+| Programming Language | Time Limit   | Memory Allocation |
+|----------------------|--------------|-------------------|
+| Java                 | Around 4 sec | Around 256 MB    |
+| C++                  | Around 4 sec | Around 256 MB    |
+| Python               | Around 8 sec | Around 512 MB    |
+| Other Accepted Languages | Varies    | Varies           |
+
+
+## Q: How can I reach out to USACO administrators or support?
+
+Contact information for USACO organizers can usually be found on their website. Look for instructions on how to contact contest organizers or support staff for assistance with any issues or inquiries.
+## Q: Who is responsible for organizing and overseeing the competition?
```

Comment:
seems pretty much the same as KTBYTE question and answer. And I don't think this is useful to include. Remove.

Reply:
True, I removed it